### PR TITLE
Fix API call for climate devices. 

### PR DESCRIPTION
**Pull request by rogerxaic**
Using floats instead of ints and changed the precision of steps to 0.5 instead of 1 as per the problems stated in this thread: https://community.home-assistant.io/t/smart-life-tuya-show-wrong-temperature/89093/69?u=rogermiret.

### CODE REVIEW

**Comment by asanchez156** [comment]
Good fix! Waiting forward to be approved this pull request!

File: setup.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import setuptools
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 setuptools.setup(
     name="tuyaha",
-    version="0.0.5",
-    author="Pavlo Annekov and original Tuya authors",
+    version="0.0.6",
```
Comment (PaulAnnekov) [inline setup.py]: I will update version in a separate commit, please, revert.

File: setup.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+    author="ThePinkOwl (Roger Miret Giné), Pavlo Annekov and original Tuya authors",
```
Comment (PaulAnnekov) [inline setup.py]: Not a good idea, imagine what a long list of contributors will be there. I think the best place is integrated in github itself, check the screenshot, you will be there with us :wink: .
image

**Comment by PaulAnnekov** [comment]
@rogerxaic sorry for delay. Thanks for your contribution ;).

**Pull request merged**